docs/kyma-environment-broker/08-06-orchestration-status.md

```diff
@@ -53,7 +50,7 @@ This tutorial shows how to check the orchestration status. Using this API, you c
 
 ## Fetch all orchestrations status
 
-Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-orchestration) to verify that the orchestration succeeded.
+Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](03-10-orchestration.md) to verify that the orchestration succeeded.
```

Comment:
I cannot see any request header details in the linked document. Did you mean `03-05-authorization.md`?

Reply:
Yes, wrong link. Thanks for spotting! 🙂